### ISSUE TITLE
Default font size to 13

### ISSUE THREAD

**Issue by Maniform** [xournalpp/xournalpp#1677]
**Affects versions :**
 - OS: Ubuntu 19.10
 - Gnome
 - Version of Xournal++ 1.1.0+dev

**Describe the bug**
DEFAUT_FONT_SIZE in file Settings.cpp is setted to 12 but this font size isn't aligned with background lines.

**Expected behavior**
Two solutions :
- set DEFAUT_FONT_SIZE to 13
- change default background

**Screenshots of Problem**
![Font size](https://user-images.githubusercontent.com/44653826/71087684-11c61e00-219d-11ea-933e-fe6b072932f5.png)

**Comment by nekohayo** [comment]
Worse, in my case, with Xournal 1.0.19 (Fedora 33 package), running under a GNOME 3.38 Xorg/X11 session, it defaults to a nonexistent/nonsensical font family and size:

![Screenshot from 2020-12-07 23-10-32](https://user-images.githubusercontent.com/479401/101439076-c0e59300-38e1-11eb-860f-06b729c11b0b.png)

**Comment by Photon89** [comment]
I'm using squared paper and there font size 11 works best (though not perfect). Maybe the background should be adjusted to fit some font size?

**Comment by rolandlo** [comment]
The squares of the graph paper are 0.5 cm in length. I don't think we should change that. You are free to change your default though (in `pagetemplates.ini`)

**Comment by Photon89** [comment]
Good point! Then maybe have some non-integer font size which fits the square size exactly?

**Comment by rolandlo** [comment]
@Maniform With correct DPI scaling (in version 1.0.x you only get that with DisplayDPI set to the default 72), the font Sans Regular in font size 20 is exactly aligned to the lines. This may be a bit large for a default.

**Comment by Photon89** [comment]
In my case it's 96 on a 12" display. I could set it to 72 and increase the system wide font size in principle...

**Comment by rolandlo** [comment]
In version 1.1.0+dev the display DPI setting (in the zoom panel of menu `edit > preferences`) doesn't affect how large fonts are (in absolute size) any more. It corresponds to the size you see for display DPI 72 in version 1.0.x.

**Comment by Photon89** [comment]
It tried to set this setting to 96 DPI (matching my setting in the system font configuration) but it does not align well to the squared  paper still: 
![Ungespeichertes Dokument - Xournal++_001](https://user-images.githubusercontent.com/1191480/109386122-f0aca180-78f8-11eb-9595-2559066774c2.png)

**Comment by rolandlo** [comment]
Sans Regular 20 should align to the Ruled background (where the lines have a distance of 24pt = 1/3 inch), not to the Graph background (where the lines have a distance of 14.17 pt = 1/2 cm). It's hard to align to both.

**Comment by Photon89** [comment]
Maybe it is possible to have a custom font size setting "align to squared paper" where the font size is, say, 11pt and the line spacing is some ugly number such that the alignment fits to squared paper?

**Comment by rolandlo** [comment]
The file format currently doesn't support a custom line spacing, I think.

**Comment by Photon89** [comment]
I see... If it is possible to implement without a huge amount of work, one could even have a checkbox "match line spacing to current background type" which calculates a line spacing based on the current DPI setting, font size and background type (would work only for the default background types shipped with Xournal++ obviously).
